views/tracks.hbs

```diff
@@ -0,0 +1,22 @@
+{{!-- tracks --}}
+<div class="selector content">
+
+    <div class="display-6 navbar bg-color text-white mb-2 p-4">
+        <p class="track-air">Title</p>
+        <p class="track-air">Listen</p>
+    </div>
+
+    {{#each tracks}}
+    <div class="bg-hover rounded navbar text-light bg-transp mb-2">
+        <p class="track-air font-weight-bold">{{this.name}}</p>
+        <figure class="track-air">
+            <audio
+                class="align-middle height-40"src="{{this.uri}}" controls="controls" type="audio/mp3">
```

Comment:
Buen trabajo Dann aquí podriamos agregar un{{#if this.uri}} para la etiqueta audio y un {{else}} para el caso no disponible